packages/utils/src/fallbackSubgraph.ts

```diff
@@ -0,0 +1,85 @@
+import { GraphQLClient } from "graphql-request";
+import { Variables, RequestDocument } from "graphql-request/dist/types";
+import { Headers } from "graphql-request/dist/types.dom";
+
+import { Logger } from "./logger";
+import { createLoggingContext } from "./request";
+
+export type SubgraphSyncRecord = {
+  client: GraphQLClient;
+  uri: string;
+  synced: boolean;
+  latestBlock: number;
+  syncedBlock: number;
+};
+
+export class FallbackSubgraph {
+  private readonly subgraphs: SubgraphSyncRecord[];
+
+  constructor(
+    private readonly logger: Logger,
+    private readonly chainId: number,
+    private readonly uris: string[],
+    private readonly allowUnsynced: number = 1,
```

Comment:
what does `allowUnsynced` mean here?

Reply:
If it's follow-on from the router/sdk configs it means the number of blocks a subgraph is allowed to be behind by before we consider it truly out of sync, and `1` is not a great default -- maybe like `25`?

Reply:
Gotcha, I actually meant to take the default out, and I'll do so now. It should come straight from config, no default should be set here.